python/create_init.py

```diff
@@ -21,6 +21,9 @@
 imports.sort()
 languages.sort()
 
+if len(languages) == 0:
```

Comment:
```suggestion
if not languages:
```

Here's the alternative if someone wants it.